src/main/java/com/snowflake/kafka/connector/internal/streaming/SnowflakeSinkServiceV2.java

```diff
@@ -122,6 +126,8 @@ public SnowflakeSinkServiceV2(
             .getClient(this.connectorConfig);
 
     this.partitionsToChannel = new HashMap<>();
+
+    this.tableName2SchemaEvolutionPermission = new HashMap<>();
```

Comment:
why not add the same logic here?

Reply:
The table2topic is empty in this function.